solvebio/resource/dataset.py

```diff
@@ -70,7 +70,8 @@ def get_or_create_by_full_path(cls, full_path, **kwargs):
         # Assert this is a dataset
         kwargs['assert_type'] = 'dataset'
         kwargs['object_type'] = 'dataset'
```

Comment:
why does it need both assert and object_type?

Reply:
`assert_type` is needed for the `get` in get_or_create and `object_type` is needed for the `create`